test/extensions/filters/http/ext_proc/matching_utils_test.cc

```diff
@@ -0,0 +1,121 @@
+#include <algorithm>
```

Comment:
very much do not need all of these imports; need to figure out which are actually needed